bool search
bool detected
int32 id
float64 pose_x
float64 pose_y